README.rst

```diff
@@ -101,43 +119,50 @@ Now go to the ``data-model`` directory and run the ``./init_qwat.sh`` script:
 
 The script has the following options:
 
-- ``-p``                   PG service to connect to the database.
+- ``-p``                     PG service to connect to the database.
 - ``-s`` or ``--srid``         PostGIS SRID. Default to 21781 (ch1903)
-- ``-d`` or ``--drop-schema``  drop schemas (cascaded) if they exist
+- ``-d`` or ``--drop-schema``    drop schemas (cascaded) if they exist
 - ``-r`` or ``--create-roles`` create roles in the database
 
-You can restore a sample dataset, then download the data sample and insert it into the qwat database:
+Optionally, You can restore a sample dataset, then download the data sample and insert it into the QWAT database:
 
 ::
 
-        QWAT_VERSION=`cat system/CURRENT_VERSION.txt`
-        # With wget
-        wget "https://github.com/qwat/qwat-data-model/releases/download/$QWAT_VERSION/qwat_v"$QWAT_VERSION"_data_only_sample.backup"
-        pg_restore -U postgres --dbname qwat -e --no-owner --verbose --jobs=3 --disable-triggers --port 5432 "qwat_v"$QWAT_VERSION"_data_only_sample.backup"
+    QWAT_VERSION=`cat system/CURRENT_VERSION.txt`
+    # With wget
+    wget -q -O qwat_dump.backup "https://github.com/qwat/qwat-data-model/releases/download/$QWAT_VERSION/qwat_v"$QWAT_VERSION"_data_only_sample.backup"
+    pg_restore -U postgres --dbname qwat -e --no-owner --verbose --jobs=3 --disable-triggers --port 5432 qwat_dump.backup
+
+Second way. Using pg_restore
+````````````````````````````
 
-        # or directly with curl
-        curl -L "https://github.com/qwat/qwat-data-model/releases/download/$VERSION/qwat_v"$VERSION"_data_only_sample.backup" | pg_restore -U postgres --dbname qwat -e --no-owner --verbose --disable-triggers --port 5432
+For now, it's not possible to restore only the structure of the database. Using this method, you will also install the example dataset.
```

Comment:
I would not add this sentence here. The user already knows that he's about to create a database populated with the sample data, so I would not confuse him talking about restoring only the structure of the database. What do you think?

Reply:
With your idea for the title. I'm agree with you.